swagger.yaml

```diff
@@ -3509,6 +3509,62 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/infra-envs/{infra_env_id}/downloads/ipxe-script:
```

Comment:
Any reason not to put it under /v2/infra-envs/{infra_env_id}/downloads/files ?

Reply:
Not really, it just didn't come to mind. Want me to change it? (I'll also update the enhancement)